src/Round.js

```diff
@@ -4,23 +4,41 @@ import Puzzle from './Puzzle.js';
 import Wheel from './Wheel.js';
 
 class Round {
-  constructor(player, puzzle, game, wheel) {
+  constructor(players, playerOne, playerTwo, playerThree, currentPlayer, puzzle, game, wheel) {
```

Comment:
We shouldn't need to pass the players individually through the constructor as we will have access to the array of players, as well as the currentPlayer.